tests/unit_tests/test_wave_03.py

```diff
@@ -130,8 +130,8 @@ def test_swap_items_from_their_empty_returns_false():
     nobodys_item = Item()
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
+    assert len(fatimah.inventory) == 3
+    assert len(jolie.inventory) == 0
+    assert not result
+    assert nobodys_item not in jolie.inventory
```

Comment:
Could also add an assertion to check that Fatimah's inventory list still looks correct: 
```python
assert itam_a in fatimah.inventory
assert itam_b in fatimah.inventory
assert itam_c in fatimah.inventory